torch/csrc/profiler/kineto_client_interface.cpp

```diff
@@ -18,16 +18,29 @@ class LibKinetoClient : public libkineto::ClientInterface {
     reportInputShapes_ = setupOpInputsCollection;
   }
 
+#ifdef USE_KINETO_UPDATED
```

Comment:
You can't reuse `USE_KINETO_UPDATED`. If you want to go this route you have to define a new variable or switch it to a version counter.

Reply:
This #ifdef is just a workround to pass CI tests (as FB internal diff contain changes on both pytorch and kineto). This #ifdef guard will be removed on submodule update. 

Last time this workaround that has successfully worked as I heard failed for some reason... so I first retry this workaround. If this is failing again, I will refactor/rewrite my changes.